smarts/sstudio/genhistories.py

```diff
@@ -277,34 +293,96 @@ def column_val_in_row(self, row, col_name: str):
                 dx = float(self._next_row["x"]) - float(row["x"])
                 dy = float(self._next_row["y"]) - float(row["y"])
                 dm = np.linalg.norm((dx, dy))
-                if dm > 0.0:
+                if dm != 0.0 and (
+                    self._heading_min_speed is None or dm > self._heading_min_speed
```

Comment:
How can `self._heading_min_speed` be `None` if its default value is 0.22?

Reply:
> How can `self._heading_min_speed` be `None` if its default value is 0.22?

Good point.  I originally hadn't given it a default value, but changed my mind when I realized they could just set it to `-1` if they didn't want to use it.  I'll update this...